chess-AI/experimental-work/output_representation.py

```diff
@@ -184,7 +181,11 @@ def convert_uci_move_to_policy_indices(self, uci_move, board_t):
                 direction = "N" if ns_value > 0 else "S"
             direction += "E" if ew_value > 0 else "W"
 
-        return np.array([*start_coords, self.codes_list.index((max(ew_value, ns_value),
+        """
+        NOTE: I added the abs value into the max function, fixed issues that we 
+        were having
+        """
+        return np.array([*start_coords, self.codes_list.index((max(abs(ew_value), abs(ns_value)),
```

Comment:
To make this more readable, what we can do is something like this (starting from assertion above)
```python
n_squares = abs(ew_value)
assert ew_value == 0 or ns_value == 0 or abs(ns_value) == n_squares
...
return np.array([*start_coords, self.codes_list.index(n_squares, direction)])
```